backend/game/cards/discard_cards/victory_point_manipulation_cards/test_drop_from_high_altitude.py

```diff
@@ -0,0 +1,15 @@
+from game.locations import Locations
+from game.newcards.discard_cards.victory_point_manipulation_cards.drop_from_high_altitude import DropFromHighAltitude
+
+
+def test_drop_from_high_altitude_adds_2_victory_points(player):
+    DropFromHighAltitude().immediate_effect(player, None)
+    assert player.victory_points == 2
+
+
+def test_drop_from_high_altitude_costs_5_energy():
+    assert DropFromHighAltitude().cost == 5
+
+
+def test_drop_from_high_altitude_location_tokyo():
+    assert DropFromHighAltitude().location == Locations.OUTSIDE
```

Comment:
Missing a test for comment above.
Also, is there a .location on the card?? Shouldn't that be player.location? And shouldn't it be == TOKYO, not outside?

Reply:
Caught that earlier and updated to value below

def test_drop_from_high_altitude_location_tokyo(player, five_players):
    DropFromHighAltitude().immediate_effect(player, five_players)
    assert player.location == Locations.TOKYO